.github/workflows/smoketest.yml

```diff
@@ -29,31 +29,32 @@ jobs:
           ref: v${{ env.LATEST_VERSION }}
           fetch-depth: 1
           fetch-tags: true
-      # The following grabs the current smoke test and deployment Justfile from the HEAD. This
-      # is to solve the chicken-and-egg. Once it's in the tagged release, skip this.
-      - name: Check out the current smoke test
-        uses: actions/checkout@v4
-        with:
-          clean: false
-          repository: TBD54566975/ftl.git
-          path: tmpsmoketest
-          ref: ${{ github.head_ref }}
-          fetch-depth: 0
-          sparse-checkout: |
-            smoketest
-            deployment
-          sparse-checkout-cone-mode: false
-      # When we remove the above step to fetch the current smoke tests and deployment Justfile
-      # into tmpsmoketest, remove the overwrite step as well.
-      - name: Replace the tagged release smoketest with the current smoketest
-        run: |
-          set -euo pipefail
-          echo "Replacing tagged release smoketest with current smoketest"
-          rm -r ./smoketest && mv ./tmpsmoketest/smoketest ./smoketest
-          echo "Replacing deployment Justfile with current Justfile"
-          cp ./tmpsmoketest/deployment/Justfile ./deployment/Justfile
-          echo "Removing tmpsmoketest"
-          rm -r ./tmpsmoketest
+      # # # These are disabled until the tagged release has the integration test changes.
```

Comment:
is this commented out because you plan on bringing it back?

Reply:
Yeah it was, until we had a tagged release with the smoketest changes. Going in a different direction now though 🙃